README.md

```diff
@@ -16,13 +16,13 @@ composer require --dev 10up/wp_mock
 
 Learn more about how to configure and how to use WP_Mock by reading [the WP_Mock documentation](https://wp-mock.gitbook.io/documentation/getting-started/introduction).
 
-## Support Level
+## Changelog
 
-**Active:** 10up and GoDaddy are actively working on this, and we expect to continue work for the foreseeable future including keeping tested up to the most recent version of WordPress.  Bug reports, feature requests, questions, and pull requests are welcome.
+A complete listing of all notable changes is documented in the [Changelog](https://github.com/10up/wp_mock/blob/trunk/CHANGELOG.md).
 
-## Changelog
+## Support Level
 
-A complete listing of all notable changes to Distributor are documented in [CHANGELOG.md](https://github.com/10up/wp_mock/blob/trunk/CHANGELOG.md).
+**Active:** [10up](https://10up.com) and [GoDaddy](https://godaddy.com) are actively working on this, and we expect to continue work for the foreseeable future including keeping tested up to the most recent version of WordPress and PHP. Bug reports, feature requests, questions, and pull requests are welcome.
```

Comment:
```suggestion
**Active:** [10up](https://10up.com) and [GoDaddy](https://godaddy.com) are actively working on this, and we expect to continue work for the foreseeable future, including testing with the most recent version of WordPress and PHP. Bug reports, feature requests, questions, and pull requests are welcome.
```

I suspect the wording was referring to "tested up to" tag, but I thought the sentence was a little awkward.